mongo.go

```diff
@@ -102,8 +122,16 @@ func (mongo mongoConfig) restore(sb s3Backend, keyPath, archive string) error {
 
 	log.Debug("Read mongo file")
 
-	key := getKey(keyPath)
-	r, err := newDecryptor(key, fr)
+	privateKey, err := getPrivateKey(privateKeyPath, c4ghPassword)
+	if err != nil {
+		log.Error("Could not retrieve private key")
+
+		return err
+	}
```

Comment:
```suggestion
	privateKey, err := getPrivateKey(privateKeyPath, c4ghPassword)
	if err != nil {
		return err
	}
```

No point of logging an error here since `err` will be propagated to main and logged as `fatal`

Reply:
same